components/ws-manager/pkg/manager/monitor.go

```diff
@@ -1284,6 +1280,14 @@ func (m *Monitor) finalizeWorkspaceContent(ctx context.Context, wso *workspaceOb
 	)
 	t := time.Now()
 
+	disposalStatus = &workspaceDisposalStatus{
+		Status: DisposalStarted,
+	}
+	err = m.manager.markDisposalStatus(ctx, workspaceID, disposalStatus)
```

Comment:
shouldn't this be done on line 1116 instead?

Reply:
this one?
https://github.com/gitpod-io/gitpod/pull/11950/files#diff-f7e47990d20d5101157fb2978f0171823bfd55a42a3ab755395b1df96911abf9R1116

Reply:
yeah. why did we need to move mark lower? :thinking: 
concerned about some unknown side effects of this.

Reply:
This is to change to a more accurate disposal start timing.